src/Microsoft.DotNet.Darc/src/DarcLib/LocalGitClient.cs

```diff
@@ -430,7 +430,7 @@ private static void CleanRepoAndSubmodules(LibGit2Sharp.Repository repo, ILogger
                         {
                             log.LogDebug($"Submodule {sub.Name} in {subRepoPath} is already initialized, trying to adopt from super-repo {repo.Info.Path}");
 
-                            // superrepo thinks it is initialized, but it's orphaned.  Go back to the master repo to find out where this is supposed to point.
+                            // superrepo thinks it is initialized, but it's orphaned.  Go back to the main repo to find out where this is supposed to point.
```

Comment:
I don't think this is referring to a branch. renaming this to main still makes sense, but this whole code block then needs to be changed, as it refers to masterRepo, masterSubModule, etc...